### PR TITLE
adds timer-driver for tim21 and tim22 (on L0)

### PR DESCRIPTION
**Pull request by shufps**
Adds timer-driver tim21 and tim22 that are available on L0

### CODE REVIEW

**Review by Dirbaio — APPROVED** [review]
thanks!

File: embassy-stm32/Cargo.toml [hunk 1]
```diff
@@ -139,6 +139,10 @@ time-driver-tim11 = ["_time-driver"]
 time-driver-tim12 = ["_time-driver"]
 ## Use TIM15 as time driver
 time-driver-tim15 = ["_time-driver"]
+## Use TIM21 as time driver
+time-driver-tim22 = ["_time-driver"]
```
Comment (Dirbaio) [inline embassy-stm32/Cargo.toml:143]: should be tim21

Comment (shufps) [reply embassy-stm32/Cargo.toml]: thx, I fixed it :see_no_evil:

**Comment by Dirbaio** [comment]
can you fix ci? running rustfmt should do it

**Comment by eZioPan** [comment]
Would you fix commit comment please? It says "adds timer-driver for tim22 and tim23 (on L0)", but it actually tim21 and tim22.

**Comment by shufps** [comment]
> Would you fix commit message please? It says "adds timer-driver for tim22 and tim23 (on L0)", but it actually tim21 and tim22.

Ups thx! I just fixed the commit message.

**Comment by eZioPan** [comment]
@shufps and there are 2 places that have extra trailing white spaces. Would you please take a look at the "ci/rustfmt" errors, and try remove the trailing white spaces?

**Pull request merged**